src/main/scala/co/datamechanics/delight/Configs.scala

```diff
@@ -46,11 +46,17 @@ object Configs {
     sparkConf.getDouble("spark.delight.waitForPendingPayloadsSleepIntervalSecs", 1).seconds
   }
 
+  def logDuration(sparkConf: SparkConf): Boolean = {
+    sparkConf.getBoolean("spark.delight.logDuration", false)
```

Comment:
nit: Would be nice to have a value for this configuration property in a Scala object.

Reply:
Do you mean the key should be in a Scala object as in the following?
```
val LOG_DURATION_KEY = "spark.delight.logDuration"
```

Reply:
Yup. Or even like they did in [Spark](https://github.com/apache/spark/blob/084d38b64ecbcaa9fac47ffca5604cf2a72936fc/core/src/main/scala/org/apache/spark/internal/config/package.scala#L41-L52)

Reply:
I understand. Right now I'll avoid relying on the `ConfigBuilder` because it has changed between Spark 2.3 and Spark 3.1.
(I'm trying to limit the reliance on all Spark code: surprisingly we've been able to make a listener that works from Spark 2.3 to 3.1 thanks to stability of the SparkListener interface).

Reply:
I'm noting down this improvement for the future.